test/controllers/lists_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
What does this Test file do, do we need it?

Reply:
no idea.. I can't remember adding that, not even created that file

Reply:
haha ok, maybe double check if it's needed?